composer.json

```diff
@@ -25,7 +25,7 @@
         "symfony/monolog-bundle"               : "^3.0",
         "symfony/polyfill-apcu"                : "^1.0",
         "symfony/swiftmailer-bundle"           : "^2.3",
-        "symfony/symfony"                      : "^3.2",
+        "symfony/symfony"                      : "dev-master",
```

Comment:
I would do the PR without xliff for now (and without changing dependencies), and add the xliff command once the demo is updated to 3.3.

Keeping the deps update in this PR will just be painful, as it will create conflicts with other PRs moving to Symfony 3.3 (we already have a PR migrating to Symfony 3.3 to use the new DI features)